2022-04-14 06:16:12,810 setting paTH TO DOWNLOAD
2022-04-14 06:16:12,811 SETTING PROFILE SETTINGS FOR FIREFOX
2022-04-14 06:16:12,846 Adding firefox profile
2022-04-14 06:16:12,846 

2022-04-14 06:16:12,847 ====== WebDriver manager ======
2022-04-14 06:16:54,931 Current firefox version is 98.0
2022-04-14 06:16:54,932 Get LATEST geckodriver version for 98.0 firefox
2022-04-14 06:16:55,564 Getting latest mozilla release info for v0.31.0
2022-04-14 06:16:55,772 Trying to download new driver from https://github.com/mozilla/geckodriver/releases/download/v0.31.0/geckodriver-v0.31.0-win64.zip
2022-04-14 06:16:56,368 Driver has been saved in cache [C:\Users\TEMP\.wdm\drivers\geckodriver\win64\v0.31.0]
2022-04-14 06:17:18,366 Snowflake Connector for Python Version: 2.5.1, Python Version: 3.8.5, Platform: Windows-2012ServerR2-6.3.9600-SP0
2022-04-14 06:17:18,369 This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
2022-04-14 06:17:18,370 Setting use_openssl_only mode to False
2022-04-14 06:17:20,933 query: [SELECT 1]
2022-04-14 06:17:21,019 query execution done
2022-04-14 06:17:21,019 query: [select * from POWERDB.PLOG.BU_CONFIG_PARAMS where PROJECT_NAME = 'ENVERUSPRT_EMA...]
2022-04-14 06:17:21,775 query execution done
2022-04-14 06:17:21,776 fetching data done
2022-04-14 06:17:21,778 closed
2022-04-14 06:17:21,835 No async queries seem to be running, deleting session
2022-04-14 06:17:21,954 Execution Started
2022-04-14 06:17:21,992 Inside remove_existing_files function
2022-04-14 06:17:25,459 Existing files removed successfully
2022-04-14 06:17:25,460 Accesing website
2022-04-14 06:17:26,799 providing id and passwords
2022-04-14 06:17:34,397 click on No Button
2022-04-14 06:17:45,592 Accessing search box
2022-04-14 06:17:57,488 setting search for only inbox
2022-04-14 06:19:32,802 Accessing search box
2022-04-14 06:19:38,047 setting search for only inbox
2022-04-14 06:21:13,336 Accessing search box
2022-04-14 06:21:18,786 setting search for only inbox
2022-04-14 06:22:54,747 Accessing search box
2022-04-14 06:23:00,310 setting search for only inbox
2022-04-14 06:24:35,895 Accessing search box
2022-04-14 06:24:41,131 setting search for only inbox
2022-04-14 06:26:16,989 Accessing search box
2022-04-14 06:26:22,385 setting search for only inbox
2022-04-14 06:26:31,614 Accessing search box
2022-04-14 06:26:38,593 Accessing search box
2022-04-14 06:26:43,596 Accessing search box
2022-04-14 06:26:48,597 Accessing search box
2022-04-14 06:26:53,598 Message: Tried to run command without establishing a connection
Traceback (most recent call last):
  File "S:\IT Dev\Testing_Environment\enverus_automation\Enverus_Script.py", line 395, in main
    login()
  File "S:\IT Dev\Testing_Environment\enverus_automation\Enverus_Script.py", line 198, in login
    raise e
  File "S:\IT Dev\Testing_Environment\enverus_automation\Enverus_Script.py", line 193, in login
    raise e
  File "S:\IT Dev\Testing_Environment\enverus_automation\Enverus_Script.py", line 182, in login
    WebDriverWait(driver, 90, poll_frequency=1).until(EC.element_to_be_clickable((By.ID, "searchBoxId-Mail"))).click()
  File "C:\Python3.8.5\production_environment\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "C:\Python3.8.5\production_environment\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 297, in __call__
    element = visibility_of_element_located(self.locator)(driver)
  File "C:\Python3.8.5\production_environment\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 128, in __call__
    return _element_if_visible(_find_element(driver, self.locator))
  File "C:\Python3.8.5\production_environment\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 415, in _find_element
    raise e
  File "C:\Python3.8.5\production_environment\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 411, in _find_element
    return driver.find_element(*by)
  File "C:\Python3.8.5\production_environment\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 976, in find_element
    return self.execute(Command.FIND_ELEMENT, {
  File "C:\Python3.8.5\production_environment\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Python3.8.5\production_environment\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: Tried to run command without establishing a connection

2022-04-14 06:26:57,131 It takes -575.1777691841125 seconds to run
